integrations/jira/.port/resources/blueprints.json

```diff
@@ -103,6 +110,13 @@
       }
     },
     "relations": {
+      "sprint": {
+        "target": "jiraSprint",
+        "title": "Sprint",
+        "description": "The Jira sprint that contains this issue",
+        "required": false,
+        "many": false
+      },
```

Comment:
I see we have a relation to the sprint but in the config, I don't see where we do the mapping